File: TypeScript/src/Cell.ts
export class Cell {
	private _isAlive: boolean;

	constructor(isAlive: boolean) {
		this._isAlive = isAlive;
	}

	get isAlive() {
		return this._isAlive;
	}
	set isAlive(isAlive: boolean) {
		this._isAlive = isAlive;
	}
}

File: Java/src/main/java/com/iamdorra/Cell.java
package com.iamdorra;

class Cell {
    private boolean isAlive;

    Cell(boolean isAlive) {

        this.isAlive = isAlive;
    }

    boolean isAlive() {
        return isAlive;
    }

    void setIsAlive(boolean b) {
        isAlive = b;
    }
}
